contrib/create-waku/package.json

```diff
@@ -1,5 +1,27 @@
 {
   "name": "create-waku",
   "version": "0.4.10",
-  "bin": "./cli.js"
+  "bin": "dist/cli.js",
+  "files": [
+    "src",
+    "dist",
+    "template"
+  ],
+  "type": "commonjs",
+  "scripts": {
+    "start": "node dist/cli.js",
+    "template": "cp -r ../../examples template/",
+    "test": "node dist/generateTemplates.test.js",
```

Comment:
How and when do I use this?

Reply:
Copy templates with ``npm run template`` before publishing the package, ``npm run start`` is just for running the cli locally.

Reply:
should we run `npm test` after `npm run template`?